.github/workflows/on-main.yml

```diff
@@ -0,0 +1,37 @@
+name: "on #main"
```

Comment:
Can we name this something like "Sizes - main"

Reply:
Yeah, I was thinking everything that needs to run on main could go in here, like eventually #main-docs-publishing, benchmark results, etc